src/institutions/ViewButton.jsx

```diff
@@ -1,25 +1,31 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Link } from 'react-router'
-import * as STATUS from '../constants/statusCodes.js'
+import {
+  CREATED,
+  PARSED_WITH_ERRORS,
+  NO_SYNTACTICAL_VALIDITY_EDITS,
+  VALIDATING,
+  VALIDATED
+} from '../constants/statusCodes.js'
 
 import './ViewButton.css'
 
 const InstitutionViewButton = ({ status, institutionId, filingPeriod }) => {
-  const code = status ? status.code : STATUS.CREATED
+  const code = status ? status.code : CREATED
   let text
 
-  if (code <= STATUS.CREATED) {
+  if (code <= CREATED) {
     text = 'Upload your file'
-  } else if (code < STATUS.PARSED_WITH_ERRORS) {
+  } else if (code < PARSED_WITH_ERRORS) {
     text = 'View upload progress'
-  } else if (code === STATUS.PARSED_WITH_ERRORS) {
+  } else if (code === PARSED_WITH_ERRORS) {
     text = 'Review formatting errors'
-  } else if (code < STATUS.SYNTACTICAL_VALIDITY_EDITS) {
+  } else if (code < NO_SYNTACTICAL_VALIDITY_EDITS) {
```

Comment:
The `NO_` status happens first but would also tell us that `VALIDATING` has stopped.